components/DateDisplay.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+export default function DateDisplay({ date }) {
+  return <DateText>{date}</DateText>;
```

Comment:
Since this component gets only one text prop, you could also think about whether to just export the styled component, instead of writing it into yet another function. In the file where you want to display the Date, you can directly use the styled component and write like so: <DateText>{date}</DateText>

Reply:
@AHen-C0de Hi Angela, just now finished, can you give me an approval?